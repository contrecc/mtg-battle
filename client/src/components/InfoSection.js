import React from 'react';

export default () => {
  return (
    <div style={{marginTop: '50px'}}>
      <hr />
      <h3>
        Site built by <a className="text-info" href="https://github.com/contrecc">Colin Contreary</a>
      </h3>
      <p>It uses React, Reactstrap, and Express</p>
      <p>
        All card data from <a className="text-info" href="https://scryfall.com">Scryfall</a>
      </p>
      <p>
        MTG Battle is unofficial Fan Content permitted under the Fan Content
        Policy. Not approved/endorsed by Wizards. Portions of the materials used
        are property of Wizards of the Coast. &copy;Wizards of the Coast LLC
      </p>
    </div>
  );
};
