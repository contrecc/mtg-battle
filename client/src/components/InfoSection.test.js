import InfoSection from './InfoSection';

describe('InfoSection', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<InfoSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
