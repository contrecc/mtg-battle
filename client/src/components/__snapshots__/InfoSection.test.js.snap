// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoSection renders correctly 1`] = `
<div
  style={
    Object {
      "marginTop": "50px",
    }
  }
>
  <hr />
  <h3>
    Site built by 
    <a
      className="text-info"
      href="https://github.com/contrecc"
    >
      Colin Contreary
    </a>
  </h3>
  <p>
    It uses React, Reactstrap, and Express
  </p>
  <p>
    All card data from 
    <a
      className="text-info"
      href="https://scryfall.com"
    >
      Scryfall
    </a>
  </p>
  <p>
    MTG Battle is unofficial Fan Content permitted under the Fan Content Policy. Not approved/endorsed by Wizards. Portions of the materials used are property of Wizards of the Coast. Â©Wizards of the Coast LLC
  </p>
</div>
`;
